README.md

```diff
@@ -1,7 +1,52 @@
 # swift-differentiation
+
+This repository is a collection of useful extensions to Swift Differentiation, the experimental Swift language feature.
```

Comment:
nit: maybe link to this here: https://github.com/differentiable-swift#meet-differentiable-swift

```suggestion
This repository is a collection of useful extensions to [Swift Differentiation](https://github.com/differentiable-swift#meet-differentiable-swift), the experimental Swift language feature.
```